components/event-publisher-proxy/pkg/handler/handlertest.go

```diff
@@ -1,5 +1,5 @@
 // Package handlertest provides utilities for Handler testing.
-package handlertest
+package handler
```

Comment:
Since you removed the handler_integration test which was the only use of the `ValidateLegacyErrorResponse` and `ValidateLegacyOkResponse`, can we remove them from this file ?